crates/iota-sdk/src/iota_client_config.rs

```diff
@@ -103,6 +107,19 @@ impl IotaClientConfig {
             self.envs.push(env)
         }
     }
+
+    /// Add a new key to the keystore, and set the resulting address as active.
+    pub fn add_key(
+        &mut self,
+        alias: Option<String>,
+        keypair: IotaKeyPair,
+    ) -> Result<(), anyhow::Error> {
+        self.keystore_mut().add_key(alias, keypair)?;
+        if self.active_address.is_none() {
+            self.set_active_address(self.keystore().addresses().last().copied());
+        }
+        Ok(())
+    }
```

Comment:
Could we turn this into a `ConfigBuilder` and have its`finish()` method do the "active address/env" switch? This way this would we wouldn't have to make those methods multi-purpose.

Reply:
Decided to just remove the fn 🤷‍♀️